tests/core/unittests/conftest.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+"""Common fixtures and utils for tests."""
+
+import os
+
+import pytest
+import yaml
+
+from metaopt.io.database import Database
+from metaopt.io.database.mongodb import MongoDB
+
+TEST_DIR = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Just to double check where does this point to?

Reply:
This is the directory where *experiment.yaml* can be found, so that it can be loaded and used for the tests.